scripts/build-markdown-pieces-gene-translate.py

```diff
@@ -141,15 +141,21 @@ def isnull(value):
             if line:
                 if line not in ref_id_list:
                     print(f"ERROR: requested input id {line} not found in ref_id_list", file=sys.stderr)
-                    sys.exit(-1)
+                    #sys.exit(-1)
```

Comment:
should this continue to be disabled?

Reply:
the goal of filtering the lists is so that you get a warning than an id was skipped rather than an error and a quit message

Reply:
sure - three thoughts:

first, if you want to remove the exit behavior, IMO you should remove the sys.exit line, not just comment it out.

second, an ERROR should result in exit. maybe this should be a WARNING?

third, the challenge with doing things this way is that the output might (will) get lost in the shuffle (all the intermediate output). we could collect these messages and output them at the end of the run; what do you think?